barcode_detection/boundingbox/bounding_box.py

```diff
@@ -0,0 +1,6 @@
+class BoundingBox:
+    def __init__(self, x: int, y: int, width: int, height: int):
+        self.x = x
```

Comment:
Not clear, what x, y describe -- center of BB? top left corner? At least some comments are required